terraform-compliance-aws/action.yml

```diff
@@ -0,0 +1,56 @@
+name: AWS Terraform Compliance
+description: Checks terraform complicance with aws
+inputs:
+  target-folder:
+    description: The target folder
+    required: true
+  subfolders:
+    description: A space delimited list of sub-folders with terragrunt.hcl files
+    required: true
+  iam-role:
+    description: The iam role to assume
+    required: true
+runs:
+  using: composite
+  steps:
+    - name: List Compliance folder
+      shell: bash
+      run: |
+        ls -la
+
+    - name: Terraform Compliance setup
+      shell: bash
+      env:
+        TARGET_FOLDER: ${{ inputs.target-folder }}
+        SUBFOLDERS: ${{ inputs.subfolders }}
+        TERRAGRUNT_IAM_ROLE: ${{ inputs.iam-role }}
+      run: |
+        cd $TARGET_FOLDER
```

Comment:
might be a personal preference but I'd probably use `working-directory` instead of an initial `cd`